### PR TITLE
Add links to VC Data Model for intro

### PR DESCRIPTION
**Pull request by brentzundel**
This PR adds a brief introduction with links to the latest VC Data Model as a resource for readers unfamiliar with the terms and concepts in the charter.

Related to Issue #59

Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-wg-charter/pull/75.html" title="Last updated on Feb 23, 2022, 4:26 PM UTC (cdc1da2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-wg-charter/75/88d0f31...brentzundel:cdc1da2.html" title="Last updated on Feb 23, 2022, 4:26 PM UTC (cdc1da2)">Diff</a>

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Review by kdenhartog — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -76,6 +76,12 @@ <h1 id="title"><i class="todo">PROPOSED</i> Verifiable Credentials Working Group
 
       <p class="mission">The <strong>mission</strong> of the <a href="">Verifiable Credentials Working Group</a> is to make expressing, exchanging, and verifying credentials easier and more secure on the web.</p>
 
+      <p class="motivation">
+Readers that are new to this work should examine the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```
Comment (iherman) [inline index.html]: Is "examine" really the right word? Maybe "consult"?

But this is coming from a non-native English speaker, so feel free to ignore.

**Review by iherman — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (msporny) [inline index.html]:
```suggestion
Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```

Comment (msporny) [reply index.html]: "read" is probably simpler and more universally understood. Suggested change below.

Comment (OR13) [inline index.html]:
```suggestion
Readers that are new to this work should review the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```

**Review by OR13 — APPROVED** [review]

**Review by msporny — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: *lol* -- I wish Github had a "you have 125 pending code reviews that you thought you had submitted, but you didn't." reminder on the UI (see above). The feature would be a huge help for me.

**Comment by Sakurann** [comment]
I was hoping this PR could clarify the previous comment wrt credential vs verifiable credential, etc. a little more clearer
https://github.com/w3c/vc-wg-charter/pull/60#discussion_r799021736

File: index.html [hunk 2]
```diff
@@ -76,6 +76,12 @@ <h1 id="title"><i class="todo">PROPOSED</i> Verifiable Credentials Working Group
 
       <p class="mission">The <strong>mission</strong> of the <a href="">Verifiable Credentials Working Group</a> is to make expressing, exchanging, and verifying credentials easier and more secure on the web.</p>
 
+      <p class="motivation">
+Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
+Credentials Data Model</a>. The <a href="https://www.w3.org/TR/vc-data-model/#introduction">Introduction</a> and
+<a href="https://www.w3.org/TR/vc-data-model/#terminology">Terminology</a> may be particularly helpful to W3C Members.
```
Comment (brentzundel) [inline index.html]:
```suggestion
<a href="https://www.w3.org/TR/vc-data-model/#terminology">Terminology</a> may be particularly helpful to W3C Members
seeking to better understand some the terminology used in this charter e.g., credentials vs. verifiable credentials.
```

**Comment by brentzundel** [comment]
> I was hoping this PR could clarify the previous comment wrt credential vs verifiable credential, etc. a little more clearer [#60 (comment)](https://github.com/w3c/vc-wg-charter/pull/60#discussion_r799021736)

@Sakurann please let me know if the changes made in [2e16fe4](https://github.com/w3c/vc-wg-charter/pull/75/commits/2e16fe4a18d944a964f8e07f1761de28f1c6b41e) address your concern.

File: index.html [hunk 3]
```diff
@@ -76,6 +76,13 @@ <h1 id="title"><i class="todo">PROPOSED</i> Verifiable Credentials Working Group
 
       <p class="mission">The <strong>mission</strong> of the <a href="">Verifiable Credentials Working Group</a> is to make expressing, exchanging, and verifying credentials easier and more secure on the web.</p>
 
+      <p class="motivation">
+Readers that are new to this work should read the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
+Credentials Data Model</a>. The <a href="https://www.w3.org/TR/vc-data-model/#introduction">Introduction</a> and
+<a href="https://www.w3.org/TR/vc-data-model/#terminology">Terminology</a> may be particularly helpful to W3C Members
+seeking to better understand some the terminology used in this charter e.g., credentials vs. verifiable credentials.
```
Comment (Sakurann) [inline index.html]:
```suggestion
seeking to better understand some the terminology used in this charter e.g., credentials vs. verifiable credentials and presentations vs. verifiable presentations.
```

**Review by Sakurann — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2022-02-23-vcwg#section4-4) on 2022-02-23

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.4. Add links to VC Data Model for intro (pr vc-wg-charter#75)

_See github pull request [vc-wg-charter#75](https://github.com/w3c/vc-wg-charter/pull/75)._




**Brent Zundel:** this PR adds direct links to existing data model text, regarding background.  
… this keeps the charter text simpler.  
… this PR helps first time readers of the charter make sense of its history.  
… i took one of the suggested changes.  

> *Kristina Yasuda:* I just made one suggestion.

> *Kristina Yasuda:* happy to approve then.

**Brent Zundel:** objections to merging?.  
… hearing no objections.  

**Manu Sporny:** great we are making progress.  

</details>

**Pull request merged**